### ISSUE TITLE
Link for ALL jpegs and ALL raws 

### ISSUE THREAD

**Issue by Evgeny2112** [schnatterer/pmcaFilesystemServer#2]
Hi. Is it possible to add a link to download all images at once, and not one at a time? Or download folder at once?

**Comment by schnatterer** [comment]
Hi, I'm convinced an experienced Java developer could implement a download as zip file in a couple of hours.
Unfortunately, I don't have a couple of hours at the moment.
I'll leave this issue open, maybe someone else is willing to do it. 

Isn't it possible to download images rather conveniently using Sony Imaging Edge?
TBH, I only implemented this to download 4K Videos, because this is not possible with Imaging Edge and my camera. Why so ever...

**Comment by Evgeny2112** [comment]
With Imaging edge i can't download raws, only jpegs.

**Comment by schnatterer** [comment]
No, really? The Sony folks seem to miss a lot of practical use cases for their cameras :-/

Seems there are more use cases for this app than I had imagined...
